runtime/curl.h

```diff
@@ -67,3 +67,29 @@ struct CurlMemoryUsage : vk::not_copyable {
 
   friend class vk::singleton<CurlMemoryUsage>;
 };
+
+class CurlRequest : private vk::not_copyable {
```

Comment:
Also let's wrap in separate namespace everything related to asynchronous curl

Reply:
done